examples/fedpca_examples/dim_reduction/client.py

```diff
@@ -0,0 +1,78 @@
+import argparse
+from pathlib import Path
+from typing import Tuple
+
+import flwr as fl
+import torch
+import torch.nn as nn
+from flwr.common import Config
+from torch.nn.modules.loss import _Loss
+from torch.optim import Optimizer
+from torch.utils.data import DataLoader
+from torchvision.transforms import transforms
+
+from examples.fedpca_examples.dim_reduction.mnist_model import MnistNet
+from fl4health.clients.basic_client import BasicClient
+from fl4health.preprocessing.pca_preprocessor import PCAPreprocessor
+from fl4health.utils.dataset import MNISTDataset
+from fl4health.utils.metrics import Accuracy
+from fl4health.utils.sampler import DirichletLabelBasedSampler
+
+
+def get_mnist_dataset(data_dir: Path, train: bool) -> MNISTDataset:
+    transform = transforms.Compose(
+        [
+            transforms.ToTensor(),
+            transforms.Normalize((0.5), (0.5)),
+        ]
+    )
+    return MNISTDataset(data_dir, train=train, transform=transform)
```

Comment:
As a convention, we've settled on acronyms in classes being upper-case for the first letter and lowercase thereafter (this is just a choice we've made, the conventions allow for either). So this would mean that"
- `PCAPreprocessor` -> `PcaPreprocessor`
- `MNISTDataset` -> `MnistDataset`
- `MnistFedPCAClient` -> `MnistFedPcaClient`
If there are other places in the code, this change should be made there too 🙂